### PR TITLE
[TASK] Color-code domains based on verification

### PR DESCRIPTION
**Pull request by wilsonianb**
DEC-530

### CODE REVIEW

File: styles/custom.css [hunk 1]
```diff
@@ -193,3 +193,11 @@ tr:hover {
 tr a:hover {
   text-decoration:underline;
 }
+
+.verified, .verified:visited {
+  color: green;
```
Comment (bsteinlo) [inline styles/custom.css]: Use hex value- -> #5ac88f

File: styles/custom.css [hunk 2]
```diff
@@ -193,3 +193,11 @@ tr:hover {
...
+}
+
+.unverified, .unverified:visited {
+  color: red;
+}
```
Comment (bsteinlo) [inline styles/custom.css:211]: #da6265

File: views/validator.html [hunk 3]
```diff
@@ -1,8 +1,11 @@
 
 <div class='row'>
   <h1><a ng-href='/#/'>Validator Registry</a></h1>
-  <h3 ng-if="domain" class='validator-show-domain'><a href='https://{{domain}}'>{{domain}}</a> - {{validatorPublicKey}}</h3>
-  <h3 ng-if="!domain" class='validator-show-domain'><a style='text-decoration:underline; font-weight:bold' href='https://ripple.com/build/rippled-apis/rippled-setup/#domain-verification' target='_blank'>Unverified</a></h3>
+  <h3 class='validator-show-domain'>
+    <a ng-if="domain" href='https://{{domain}}' class='verified'>{{domain}}</a>
```
Comment (bsteinlo) [inline views/validator.html:5]: I believe you can combine this into a tertiary statement--- something like `unverified': !isVerified(), verified : isVerified()` 

and then check it with something you can test against `this.isLoggedIn = function() {
      return....;
    };`

File: views/validators.html [hunk 4]
```diff
@@ -42,8 +42,8 @@ <h3 ng-if="ValidatorSelector.getSelectedValidators().length > 0">
       <td>{{validator.agreement_coefficient.toFixed(5)}}</td>
       <td>{{validator.disagreement_coefficient.toFixed(5)}}</td>
       <td>{{validator.validations}}</td>
-      <td ng-if="validator.domain"><a ng-click='$event.stopPropagation()' href='https://{{validator.domain}}'>{{validator.domain}}</a></td>
-      <td ng-if="!validator.domain"><a ng-click='$event.stopPropagation()' target='_blank' href='https://ripple.com/build/rippled-apis/rippled-setup/#domain-verification'>?</a></td>
+      <td ng-if="validator.domain"><a ng-click='$event.stopPropagation()' href='https://{{validator.domain}}' class='verified'>{{validator.domain}}</a></td>
+      <td ng-if="!validator.domain"><a ng-click='$event.stopPropagation()' href='https://ripple.com/build/rippled-apis/rippled-setup/#domain-verification' class='unverified'>unverified</a></td>
```
Comment (bsteinlo) [inline views/validators.html:47]: same thing here

File: views/validators.html [hunk 5]
```diff
@@ -72,8 +72,8 @@ <h3 class='pull-left' ng-if="ValidatorSelector.getSelectedValidators().length ==
...
```
Comment (bsteinlo) [inline views/validators.html:78]: again here, can you ng-show and ng-hide as well

**Comment by wilsonianb** [comment]
Updated colors. Foregoing ternary statements for now.

**Comment by bsteinlo** [comment]
LGTM

**Pull request merged**